sections/memory_safety_in_rust.tex

```diff
@@ -35,6 +32,29 @@ \section{Memory Safety in Rust}
 
 \end{frame}
 
+
+%%%
+
+\begin{frame}[fragile]{Ergonomic scaffolding with \texttt{unwrap()}}
```

Comment:
I would rather use `.expect("message")` because the custom error message often helps. Or at least show both.